packages/faucet/src/database-helper.ts

```diff
@@ -116,17 +120,13 @@ function buildHandleInvite(request: RequestRecord, snap: DataSnapshot, config: N
       config.goldTokenAddress
     )
     const { address: tempAddress, inviteCode } = generateInviteCode()
-    const goldTx = await celo.transferGold(tempAddress, config.inviteGoldAmount)
-    const goldTxHash = await goldTx.getHash()
-    console.info(`req(${snap.key}): Gold Transaction Sent. txhash:${goldTxHash}`)
-    await snap.ref.update({ goldTxHash })
-    await goldTx.waitReceipt()
-
-    const dollarTx = await celo.transferDollars(tempAddress, config.inviteDollarAmount)
-    const dollarTxHash = await dollarTx.getHash()
-    console.info(`req(${snap.key}): Dollar Transaction Sent. txhash:${dollarTxHash}`)
-    await snap.ref.update({ dollarTxHash })
-    await dollarTx.waitReceipt()
+    // Dont need gold as fees are paid in dollars
```

Comment:
for this to work you need to specify the `GasCurrency` when sending the TX. So probably this is not working

Reply:
ah ok